appinfo/routes.php

```diff
@@ -24,6 +24,17 @@
 		['name' => 'SpeechToText#getResultPage', 'url' => '/stt/result-page/{metaTaskId}', 'verb' => 'GET'],
 
 		['name' => 'preview#getFileImage', 'url' => '/preview', 'verb' => 'GET'],
+
+		['name' => 'chattyLLM#newSession', 'url' => '/chat/new_session', 'verb' => 'PUT'],
+		['name' => 'chattyLLM#updateSession', 'url' => '/chat/update_session', 'verb' => 'PATCH'],
+		['name' => 'chattyLLM#deleteSession', 'url' => '/chat/delete_session', 'verb' => 'DELETE'],
+		['name' => 'chattyLLM#getSessions', 'url' => '/chat/sessions', 'verb' => 'GET'],
+		['name' => 'chattyLLM#newMessage', 'url' => '/chat/new_message', 'verb' => 'PUT'],
+		['name' => 'chattyLLM#deleteMessage', 'url' => '/chat/delete_message', 'verb' => 'DELETE'],
+		['name' => 'chattyLLM#getMessages', 'url' => '/chat/messages', 'verb' => 'GET'],
+		['name' => 'chattyLLM#generate', 'url' => '/chat/generate', 'verb' => 'GET'],
+		['name' => 'chattyLLM#regenerate', 'url' => '/chat/regenerate', 'verb' => 'GET'],
+		['name' => 'chattyLLM#generateTitle', 'url' => '/chat/generate_title', 'verb' => 'GET'],
```

Comment:
At some point it would be nice to migrate to the new Route annotation  by kate

Reply:
We would be losing support for the previous server versions if we do that now. Also, personally I would like to keep this alongside that, if possible, to have a quick look at all the endpoints declared in the app.